daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/ledgerinteraction/GrpcLedgerClient.scala

```diff
@@ -376,8 +376,10 @@ class GrpcLedgerClient(val grpcClient: LedgerClient, val applicationId: Applicat
       ec: ExecutionContext,
       esf: ExecutionSequencerFactory,
       mat: Materializer,
-  ): Future[List[User]] =
-    grpcClient.userManagementClient.listUsers().map(_.toList)
+  ): Future[List[User]] = {
+    // TODO participant user management: Emulating no-pagination
+    grpcClient.userManagementClient.listUsers(pageToken = "", pageSize = 10000).map(_.toList)
```

Comment:
@victormueller-da @moritzkiefer-da @nickchapman-da Currently requesting fixed large enough pageSize hoping to retrieve all the users. Not sure whether this is good enough for this PR.

Reply:
Thanks @cocreature!